bin/index.js

```diff
@@ -8,9 +8,11 @@ const ora = require('ora');
 const getPort = require('get-port');
 const inquirer = require('inquirer');
 const opn = require('opn');
-const minimist = require('minimist');
```

Comment:
now that we're no longer using minimist, can we remove it from package.json dependencies?

Reply:
I checked it and found that it is being used in the `server/index.js` file.